lib/widgets/explore/product_item.dart

```diff
@@ -1,15 +1,57 @@
 import 'package:flutter/material.dart';
 
 class ProductItem extends StatelessWidget {
-  String title;
+  // TODO: Image;
+  final String title;
+  final String brand;
+  final String size; //? Should this be int or String?
+  //? Sizes could be either "small" or "36"
 
-  ProductItem(this.title);
+  ProductItem(
+      {@required this.title, @required this.brand, @required this.size});
```

Comment:
Add a ',' after size.

Nice to add the required tag. We need to be better at adding these tags for named args.